rust/abacus-base/src/types/checkpoint_syncer.rs

```diff
@@ -77,20 +78,25 @@ pub enum CheckpointSyncers {
 
 #[async_trait]
 impl CheckpointSyncer for CheckpointSyncers {
+    #[instrument(err, skip(self), level = "debug")]
```

Comment:
ah doing it at the `CheckpointSyncers` level is much nicer than what I did [here](https://github.com/abacus-network/abacus-monorepo/pull/361/files#diff-b5624ab7cd098615872f3a72f5b36d769ef28e9ee27d6f748b3edffde86376b9R48), i'm gonna remove in favor of this